src/DxfParser.ts

```diff
@@ -587,15 +587,11 @@ export default class DxfParser {
 						viewPort.ucsYAxis = parsePoint(curr);
 						curr = scanner.next();
 						break;
-					case 110:
-						viewPort.ucsOrigin = parsePoint(curr);
-						curr = scanner.next();
-						break;
 					case 281:
 						viewPort.renderMode = curr.value as string;
 						curr = scanner.next();
 						break;
-					case 281:
+					case 282:
```

Comment:
case 281 was above. By inspection of http://docs.autodesk.com/ACD/2011/ENU/filesDXF/WS1a9193826455f5ff18cb41610ec0a2e719-79c6.htm, this appears to be case 282